src/OffensiveWordSearcher.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Pinnacle\CommonValueObjects;
+
+use UnexpectedValueException;
+
+/**
+ * Class OffensiveWordSearcher.
+ */
+class OffensiveWordSearcher
+{
+    /**
+     * @var string Path to offensive words file.
+     */
+    const OFFENSIVE_WORDS_FILE_PATH = 'resources/offensive-words.txt';
```

Comment:
I would use `__DIR__ . '/resources/offensive-words.txt` so that this is an absolute path instead of a relative path. This way issues with the current working directory can be avoided.

Reply:
Also, can we use `private const`? I don't know if that's available in PHP 7.0.

Reply:
No unfortunately it's not.